services/core-web/src/components/Forms/permits/SitePropertiesForm.js

```diff
@@ -0,0 +1,195 @@
+import React, { Component } from "react";
+import { PropTypes } from "prop-types";
+import { Row, Col, Popconfirm, Button } from "antd";
+import { Field, formValueSelector, FormSection, reduxForm, Form } from "redux-form";
+import { connect } from "react-redux";
+import { compose } from "redux";
+import RenderMultiSelect from "@/components/common/RenderMultiSelect";
+import RenderSelect from "@/components/common/RenderSelect";
+import CustomPropTypes from "@/customPropTypes";
+import { requiredList, maxLength, required } from "@common/utils/Validate";
+import {
+  getConditionalDisturbanceOptionsHash,
+  getConditionalCommodityOptions,
+  getMineTenureTypeDropdownOptions,
+  getExemptionFeeSatusDropDownOptions,
+} from "@common/selectors/staticContentSelectors";
+import { determineInspectionFeeStatus } from "@common/utils/helpers";
+import * as FORM from "@/constants/forms";
+import RenderAutoSizeField from "@/components/common/RenderAutoSizeField";
+/**
+ * @constant SitePropertiesForm renders edit/view for the NoW Application review step
+ */
+
+const propTypes = {
+  permit: CustomPropTypes.permit.isRequired,
+  mineTenureTypes: PropTypes.objectOf(CustomPropTypes.options).isRequired,
+  conditionalCommodityOptions: PropTypes.objectOf(CustomPropTypes.options).isRequired,
+  conditionalDisturbanceOptions: PropTypes.objectOf(CustomPropTypes.options).isRequired,
+  site_properties: PropTypes.objectOf(
+    PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)])
+  ).isRequired,
+  exemptionFeeSatusDropDownOptions: PropTypes.objectOf(CustomPropTypes.options).isRequired,
+  submitting: PropTypes.bool.isRequired,
+  closeModal: PropTypes.func.isRequired,
+  handleSubmit: PropTypes.func.isRequired,
+  change: PropTypes.func.isRequired,
+};
+
+const mapApplicationTypeToTenureType = (permitPrefix) =>
+  ({
+    P: ["PLR"],
+    C: ["COL"],
+    M: ["MIN"],
+    G: ["BCL", "PRL"],
+    Q: ["BCL", "PRL", "MIN"],
+  }[permitPrefix]);
+export class SitePropertiesForm extends Component {
+  componentWillReceiveProps = (nextProps) => {
+    const permitIsExploration = this.props.permit.permit_no.charAt(1) === "X";
+    const permitTypeCode = this.props.permit.permit_no.charAt(0);
```

Comment:
We should call this `permitPrefix` for consistency?

Reply:
ahh, if it's a hybrid property I can reference it via the data... let me update that everywhere.